array.js

```diff
@@ -0,0 +1,601 @@
+// // Давайте произведём 5 операций с массивом.
+// // Создайте массив styles с элементами «Джаз» и «Блюз».
+// // Добавьте «Рок-н-ролл» в конец.
+// // Замените значение в середине на «Классика». Ваш код для поиска значения в середине должен работать для массивов с любой длиной.
+// // Удалите первый элемент массива и покажите его.
+// // Вставьте Рэп и Регги в начало массива.
+
+let styles = ["Jaz", "Bluz"];
+styles.push("Rock");
+console.log(styles);
+
+styles[Math.floor(styles.length / 2)] = "Classic";
+console.log(styles);
+
+styles.shift();
+console.log(styles);
+
+styles.unshift("Rap and Reggy");
+console.log(styles);
+
+// Напишите функцию sumInput(), которая:
+// Просит пользователя ввести значения, используя prompt и сохраняет их в массив.
+// Заканчивает запрашивать значения, когда пользователь введёт не числовое значение, пустую строку или нажмёт «Отмена».
+// Подсчитывает и возвращает сумму элементов массива.
+// P.S. Ноль 0 – считается числом, не останавливайте ввод значений при вводе «0».
+
+
+function sumInput() {
+    let num = [];
+    while (true) {
```

Comment:
why do you need it here while true?
this is an infinite loop

Reply:
we discussed it. the reason of using while(true) is adding the break in the cicle